editor/src/components/canvas/controls/select-mode-control-container.tsx

```diff
@@ -148,22 +148,8 @@ export class SelectModeControlContainer extends React.Component<
   onContextMenu = (event: React.MouseEvent<HTMLDivElement>) => {
     event.stopPropagation()
     event.preventDefault()
-    const elementsUnderCursor = getAllTargetsAtPoint(
```

Comment:
so we were only using this `elementsUnderCursor` to tell if something should be hidden or not? :O I thought this was also populating the list too!

Reply:
the list contains all element path!